### PR TITLE
add ROSA variables to scale script

### PR DESCRIPTION
**Pull request by morenod**
Add ROSA related vars to create benchmark file with ROSA related parametrs

### CODE REVIEW

File: workloads/scale-perf/README.md [hunk 1]
```diff
@@ -56,21 +56,37 @@ Default: `false`
 Enable/Disable the ability to compare two runs. If set to `true`, the next set of environment variables pertaining to the type of test are required
 
 ### BASELINE_CLOUD_NAME
-Default: ``
```
Comment (dry923) [inline workloads/scale-perf/README.md:59]: @morenod can we leave the default values or was there a reson you removed them?

File: workloads/scale-perf/run_scale_fromgit.sh [hunk 2]
```diff
@@ -54,6 +69,7 @@ spec:
       serviceaccount: scaler
       poll_interval: $_poll_interval
       post_sleep: $_post_sleep
+$(_add_rosa_params)
```
Comment (rsevilla87) [inline workloads/scale-perf/run_scale_fromgit.sh:72]: This is a bit tricky, I think you should consider using an if else logic to create the Benchmark CRDs

File: workloads/scale-perf/README.md (hunk 1 above)
Comment (morenod) [reply workloads/scale-perf/README.md:59]: Empty '' is breaking the .md file generation. If default is none, there is no default

File: workloads/scale-perf/run_scale_fromgit.sh (hunk 2 above)
Comment (morenod) [reply workloads/scale-perf/run_scale_fromgit.sh:72]: It is the only way to add conditionals to a cat EOF structure without replicating all the content

File: workloads/scale-perf/README.md (hunk 1 above)
Comment (morenod) [reply workloads/scale-perf/README.md:59]: restored empty defaults

File: workloads/scale-perf/run_scale_fromgit.sh (hunk 2 above)
Comment (morenod) [reply workloads/scale-perf/run_scale_fromgit.sh]: what do you think now?

File: workloads/scale-perf/README.md [hunk 3]
```diff
@@ -53,24 +60,59 @@ Enable/Disable collection of metadata
...
+Default: ''
+
+Environment where cluster is deployed (Production, Staging)
+
+### ROSA_TOKEN
+Default: ''
+
+Token used to login on ROSA
+
+### AWS_ACCESS_KEY_ID & AWS_SECRET_ACCESS_KEY
```
Comment (dry923) [inline workloads/scale-perf/README.md:106]: Extremely minor nit but can you separate these 2 into separate items? I had to re-read it 3 times before i realized how you wanted the key and id lol.

File: workloads/scale-perf/run_scale_fromgit.sh [hunk 4]
```diff
@@ -16,76 +17,50 @@ for x in $(seq 1 $_runs); do
         exit 1
       fi
     fi
-    
-    
+
+
```
Comment (dry923) [inline workloads/scale-perf/run_scale_fromgit.sh:21]: Bonus empty space

File: workloads/scale-perf/README.md (hunk 3 above)
Comment (morenod) [reply workloads/scale-perf/README.md]: done

File: workloads/scale-perf/run_scale_fromgit.sh (hunk 4 above)
Comment (morenod) [reply workloads/scale-perf/run_scale_fromgit.sh]: done

**Review by dry923 — APPROVED** [review]
LGTM! Ignoring the router failure since this does not touch that.

**Pull request merged**